ode.php

```diff
@@ -482,71 +528,73 @@ function checkValidEmails() {
     $civiVersion = CRM_Core_BAO_Domain::version();
   }
 
-  $error = array();
+  $error = [];
 
-  $links = array(
+  $links = [
     'Contribution Page(s)' => 'civicrm/admin/contribute/thankyou',
     'Event(s)' => 'civicrm/event/manage/registration',
     'Schedule Reminder(s)' => 'civicrm/admin/scheduleReminders',
-  );
-
-  // Contribution Pages.
-  $contributionPageParams = array(
-    'is_email_receipt' => 1,
-    'receipt_from_email' => array("NOT LIKE" => "'%{$getHostName}'"),
```

Comment:
This actually turned out to be the issue on ACOP it seemed to be passing through the quote as part of the string to check